velox/functions/prestosql/DateTimeFunctions.h

```diff
@@ -1937,7 +1937,10 @@ struct ParseDurationFunction {
     re2::StringPiece valueStr;
     re2::StringPiece unit;
     if (!RE2::FullMatch(
-            std::string_view{amountUnit}, *durationRegex_, &valueStr, &unit)) {
+            absl::string_view(amountUnit.data(), amountUnit.size()),
```

Comment:
PR title says "Use re2::StringPiece", but the code uses "absl::string_view". Other code (2 lines above) uses re2::StringPiece. It would be nice to limit the number of different string views used in this code.

Reply:
Sorry. I forgot to update PR title/description.

I've replaced all `re2::StringPiece` with `absl::string_view`.

Reply:
Oh... Let's use `re2::StringPiece` for now because:
* RE2 will use `std::string_view` not `absl::string_view` eventually because RE2 migrated to `std::optional` from `absl::optional`: https://github.com/google/re2/commit/2a029e28d3080e4718581649ba9faad5347c3ff9
* `re2::StringPiece` works with old RE2 and new RE2

We can revisit this when RE2 migrates to `std::string_view` from `absl::string_view`.